api/router/BulkUpdateRouter.go

```diff
@@ -0,0 +1,26 @@
+package router
+
+import (
+	"github.com/devtron-labs/devtron/api/restHandler"
+	"github.com/gorilla/mux"
+)
+
+type BulkUpdateRouter interface {
+	initBulkUpdateRouter(bulkRouter *mux.Router)
+}
+
+type BulkUpdateRouterImpl struct {
+	restHandler restHandler.BulkUpdateRestHandler
+}
+
+func NewBulkUpdateRouterImpl(handler restHandler.BulkUpdateRestHandler) *BulkUpdateRouterImpl {
+	router := &BulkUpdateRouterImpl{
+		restHandler: handler,
+	}
+	return router
+}
+func (router BulkUpdateRouterImpl) initBulkUpdateRouter(bulkRouter *mux.Router) {
+	bulkRouter.Path("/application/see-example").HandlerFunc(router.restHandler.GetExampleOperationBulkUpdate).Methods("GET")
+	bulkRouter.Path("/application/{dryrun}").HandlerFunc(router.restHandler.GetAppNameDeploymentTemplate).Methods("POST")
```

Comment:
dryrun is not path variable it is path it should be /v1beta1/application/dryrun

Reply:
changed